site/docs/v1/tech/reference/limitations.adoc

```diff
@@ -109,6 +110,12 @@ include::docs/v1/tech/installation-guide/_downtime-upgrade-limitation.adoc[]
 
 include::docs/v1/tech/reference/_configuration-limits.adoc[]
 
+== Password Hashes
+
+There is no FusionAuth API allowing user password hashes to be exported.
+
+If you need to migrate hashes from FusionAuth to any other system, use a database export.
```

Comment:
Maybe something more targeted?
> If you are migrating off of FusionAuth Cloud, and need access to your password hashes, we can provide a database export.

Reply:
I wanted to make it applicable to cloud or self hosted solutions.